projects/libical/build.sh

```diff
@@ -0,0 +1,8 @@
+cd $SRC
+cd libical
```

Comment:
This shouldn't be needed, your Dockerfile neds in `WORKDIR libical` so you're already in here.